codehawk.properties

```diff
@@ -0,0 +1,14 @@
+category=External Analysers
+description=Analyze Java code smell
+homepageUrl=https://github.com/SDPMLab/CodeHawk
+archivedVersions=1.0
+publicVersions=1.0
+
+defaults.mavenGroupId= org.codehawk.plugin
+defaults.mavenArtifactId= codehawk
+
+1.0.description= Initial version compatible with SonarQube 7.7
+1.0.sqVersions=[7.7,LATEST]
```

Comment:
This should start at 7.9, the oldest version listed in the Marketplace

Reply:
Done, but our plugin still need fixing.